threedi_models_and_simulations/widgets/simulation_wizard.py

```diff
@@ -642,6 +644,58 @@ def setup_initial_conditions(self):
             error_msg = f"Error: {e}"
             self.parent_page.parent_wizard.plugin_dock.communication.bar_error(error_msg, log_text_color=QColor(Qt.red))
 
+    def load_csv(self):
```

Comment:
It looks like a method name is quite generic and missleading as it's dedicated for loading 1D initial water level data. Please rename it to something like `load_1d_initial_waterlevel_csv`.